### ISSUE TITLE
Create bills list view (rows + section headers)

### ISSUE THREAD

**Issue by levvaa** [ios-course/link-team-project#43]
Part of the issue #3
Depends on issue #42.

## AC:
<img width="230" alt="Screenshot 2023-05-23 at 12 32 43" src="https://github.com/ios-course/link-team-project/assets/120186417/2dadef7f-c354-4aab-a934-8f67f759100e">

## User's perspective:
- we should display a list of bills that I can scroll vertically. 
- bills should be sorted. Newer bills should be higher in the list
- bills must be grouped by date.
- each date is a heading. Under this heading, bills created on this date are displayed.
- the list of bills within a specific date is not ordered.
- the section header sticks to the top of the list while the current date accounts are being displayed.

## Implementation details:
- current data is hardcoded in memory.
- date inside header is located by default on the left.
- if the date is too long, cut off the end of the date, the line should always be one.
- for date font size 'Body' (see fonts in [wiki](https://github.com/ios-course/link-team-project/wiki/App-style-guidelines)).
- background color for header is not needed.
- format for dummy view model:
```swift
[
Date1: [Bill1, Bill2, Bill3],
Date2: [Bill1, Bill2],
Date3: [Bill1, Bill2, Bill3, Bill4],
]
```

**Comment by kirpichonok** [comment]
- We need an image of the done view.
- What should we display when no bills in the list? You can create a separate issue for the addition of some picture of an empty screen and link to that issue. At this point, you can just make it empty. 
- We can replace last bullet point with  *"The view should be scrollable vertically*"
- Specify that for displaying one bill in the list should be used `BillRowView`. 
- What fonts should be for dates? Body or Title?

**Comment by kirpichonok** [comment]
- We need an image of the completed view.
- What should we display when there are no bills in the list? You can create a separate issue for the addition of a picture of an empty screen and link to that issue. At this point, you can just make it empty. 
- We can replace the last bullet point with  *"The view should be scrollable vertically*"
- Specify that for displaying one bill in the list should be used `BillRowView`. 
- What fonts should be used for dates? Body or Title?

**Comment by kirpichonok** [comment]
- We need an image of the completed view.
- What should we display when there are no bills in the list? You can create a separate issue for the addition of a picture of an empty screen and link to that issue. At this point, you can just make it empty. 
- We can replace the last bullet point with  *"The view should be scrollable vertically*"
- Specify that for displaying one bill in the list should be used `BillRowView`. 
- What fonts should be used for dates? Body or Title?
